src/app/store/submitted-transactions/submitted-transactions.slice.ts

```diff
@@ -0,0 +1,22 @@
+import { createEntityAdapter, createSlice, EntityId, PayloadAction } from '@reduxjs/toolkit';
+
+export type SubmittedTransaction = { rawTx: string; txId: string };
```

Comment:
```suggestion
export interface SubmittedTransaction { rawTx: string; txId: string };
```

unless using the features of type, we should always default to interface, so that use of type communicates something more complex/a composition of other types.

This could also be reused in the component that takes these same props

Reply:
Ok, I was following the `createEntityAdapter` docs and they use a type in the example.

`type Book = { bookId: string; title: string }`

Reply:
The reality is that either can be used, and it comes down to some minor differences in behaviour and personal pref

Reply:
Ok, thanks, I haven't been super clear on the difference tbh so was reading up yest on it before I used type from the example. What I found wasn't super helpful.